app.py

```diff
@@ -156,6 +157,14 @@ def carstate(shortuuid):
 
     if result:
         if result.expiry > time.time():
+            # Update Attribute for car_id if in DB result else use ENV variable - only if MultiCar is enabled
+            if BACKEND_PROVIDER_MULTICAR == 'True':
+                if result.carid:
+                    new_car_id = result.carid
+                else:
+                    new_car_id = BACKEND_PROVIDER_CAR_ID
+            BackendProviderFactory.provider.car_id = new_car_id
```

Comment:
This will not work because the backend does not provide any "car_id" entity at the runtime, it is only used for the construction of the GET request URL.